src/GitUrlTagger.ts

```diff
@@ -5,20 +5,32 @@ import { IConstruct } from 'constructs';
 
 export interface GitUrlTaggerProps {
   /**
-     * The Tag key/name to use
-     *
-     * @default 'GitUrl'
-     */
+   * The Tag key/name to use
+   *
+   * @default 'GitUrl'
+   */
   readonly tagName?: string;
+
+  /**
+   * A flag on whether to try to normalize the URL found in the git config
+   * If enabled, it will turn ssh urls into https urls.
+   *
+   * @default false
+   */
+  readonly normalizeUrl?: boolean;
```

Comment:
Consider defaulting to true, as the UX is probably better with HTTPS urls vs. SSH urls

Reply:
I agree, changing it now...